tests/cnf/core/network/sriov/tests/qinq.go

```diff
@@ -956,16 +966,15 @@ func defineAndCreateClientDPDKPod(
 	Expect(err).ToNot(HaveOccurred(), "Fail to define client dpdk container")
 
 	dpdkPod, err := pod.NewBuilder(APIClient, podName, tsparams.TestNamespaceName,
-		NetConfig.DpdkTestContainer).WithSecondaryNetwork(serverPodNetConfig).
+		NetConfig.DpdkTestContainer).WithSecondaryNetwork(serverPodNetConfig).WithPrivilegedFlag().
```

Comment:
Why do you need WithPrivilegedFlag() if you created corev1.SecurityContext?

Reply:
removed